from0k2bp.tex

```diff
@@ -1194,30 +1223,42 @@
 coefficient of $G$, using the binding property observe that that must be
 the opening (and making use of the ability to cross multiply by an
 inverse scalar):
+\begin{align}
+& t_z H + \left(\textbf{f}_x \cdot \textbf{f}_y \right) G = e^2 C_z + e C_1 + C_0 
+  \label{inner_proof_product_knowledge_soundness_Cz:1} \\
+& t_z' H + \left(\textbf{f}_x' \cdot \textbf{f}_y' \right) G = e'^2 C_z + e' C_1 + C_0  
+\label{inner_proof_product_knowledge_soundness_Cz:2}
+\end{align}
+Let us remind definitions:
```

Comment:
Good idea to review definitions, but "Let us remind ourselves of the definitions", or perhaps just a simpler "Remember the definitions:" ("Let us remind definitions" is not correct)